config_test.go

```diff
@@ -272,6 +272,24 @@ func TestProducerConfigValidates(t *testing.T) {
 				cfg.Producer.RequiredAcks = WaitForAll
 			},
 			"Idempotent producer requires Net.MaxOpenRequests to be 1"},
+		{"ReadCommitted Version",
```

Comment:
These tests are testing the consumer configuration and they are written in `TestProducerConfigValidates`. IMHO it would be better to create a new test method named `TestConsumerConfigValidates` to test these settings.